pennylane_lightning/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_Generator.cpp

```diff
@@ -77,9 +77,10 @@ TEMPLATE_TEST_CASE("StateVectorKokkosManaged::applyGeneratorPhaseShift",
             kokkos_gate_svp.HostToDevice(ini_st.data(), ini_st.size());
             kokkos_gate_svm.HostToDevice(ini_st.data(), ini_st.size());
 
-            auto scale = kokkos_gntr_sv.applyGeneratorPhaseShift({1}, false);
-            kokkos_gate_svp.applyPhaseShift({1}, false, {ep});
-            kokkos_gate_svm.applyPhaseShift({1}, false, {-ep});
+            auto scale =
+                kokkos_gntr_sv.applyGenerator("PhaseShift", {1}, false);
+            kokkos_gate_svp.applyOperation("PhaseShift", {1}, false, {ep});
+            kokkos_gate_svm.applyOperation("PhaseShift", {1}, false, {-ep});
```

Comment:
Is there a new idiomatic way to directly apply the gate without a lookup?

Reply:
No, the only place such calls appeared was in the C++ tests so I would remove them.